app/code/Magento/OfflinePayments/Test/Unit/Block/Info/CheckmoTest.php

```diff
@@ -3,22 +3,24 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
```

Comment:
You can introduce `declare(strict_types=1);`

Reply:
done.